### ISSUE TITLE
Launching throws "Assertion '__n < this->size()' failed", then abort and core dump.

### ISSUE THREAD

**Issue by notkirito** [dpayne/cli-visualizer#183]
# Description
The exact error: `/usr/include/c++/11.1.0/bits/stl_vector.h:1045: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](std::vector<_Tp, _Alloc>::size_type) [with _Tp = vis::ColorDefinition; _Alloc = std::allocator<vis::ColorDefinition>; std::vector<_Tp, _Alloc>::reference = vis::ColorDefinition&; std::vector<_Tp, _Alloc>::size_type = long unsigned int]: Assertion '__n < this->size()' failed.`

It then crashes: `[1]    325436 abort (core dumped)  vis`

**Steps to reproduce**
- Install Arch Linux and `yay`
- `yay -S cli-visualizer`
- `vis`

**Comment by gibranlp** [comment]
i am having this very same issue, i used 

- paru -S cli-visualizer
- vis

**Comment by notkirito** [comment]
The AUR package is outdated, I’ll try to compile it myself.

**Comment by ZGFabian** [comment]
cli-visualizer-git package produces the same error

**Comment by GKart** [comment]
same problem

**Comment by gibranlp** [comment]
if you uninstall the AUR version, and then install it from source fixes the problem, for now,

**Comment by GKart** [comment]
Thank you very much although it should be clarified that the default settings do not work correctly but putting some custom settings works, thank you very much.

**Comment by cyberpunkrocker-zero** [comment]
Im still getting the assertion error and core dump on Arch Linux even after fixing the PKGBUILD of cli-visualizer-git to work like the 'install.sh' script.

@dpayne if I'm not mistaken, you seem to be also resposible for the AUR packages of cli-visualizer. Please update and fix them, they're both broken ('build' directory missing)

**Comment by Nils75owo** [comment]
I edited the ~/.config/vis/config file and uncommented this line:
```visualizer.fps=20```
(should be the second line)
that fixed the problem for me

**Comment by simpleauthority** [comment]
Having the same error - uncommenting the fps setting did not do anything.

**Comment by simpleauthority** [comment]
Yup. Something is wrong with the AUR package - even the git one. Just clone from GH and `./install.sh`. It works.
